runtime/tests/third_party/dummy_device.cpp

```diff
@@ -42,13 +42,18 @@ struct DummyDevice final : public Catalyst::Runtime::QuantumDevice {
     void StartTapeRecording() override {}
     void StopTapeRecording() override {}
     void PrintState() override {}
-    void NamedOperation(const std::string &, const std::vector<double> &,
```

Comment:
Don't forget to update `custom_device.rst` in docs too :)

Reply:
Indeed, thanks. Done.